scripts/startapp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
A small dev experience improvement here: I added a script that boots the API and docs/login site in one command. This will start both servers and kill them both on a Ctrl+C.